runtime/java_test.go

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package runtime
+
+import (
+	"testing"
+
+	. "github.com/mikelangelo-project/capstan/testing"
```

Comment:
Yes. Similarly as we import gocheck directly, we want to import our extenstions to it. Looking at expectations:
```go
// Expectations.
c.Assert(err, IsNil)
c.Check(boot, BootCmdEquals, args.expectedBoot, args.expectedEnv)
```
I find it better to use checkers directly (`IsNil`, `BootCmdEquals`) rather than with package name (`IsNil`, `testing.BootCmdEquals`). Would you prefer the explicit way?

I find it unpleasant, however, that Sublime is unable to "jump to definition" in case of direct import...